lib/resources/docker.rb

```diff
@@ -203,8 +203,13 @@ def parse_json_command(labels, subcommand)
         # does not include a slash since that is not a valid character in a container name
         j['names'] = j['names'].split(',').find { |c| !c.include?('/') } if j.key?('names')
 
+        # Split labels on ','
+        # This allows `its('labels') { should include 'foo=bar' }` to work
+        j['labels'] = j['labels'].split(',') if j.key?('labels')
```

Comment:
Not sure if I understand the code flow here, so this may not be a concern:

Is it possible for the JSON data to not contain the key 'labels', as the conditional suggests?  In that case, how will your users expect the `labels` field to behave in the InSpec resource?  As-is, I suspect that the `labels column will end up with nil values, so 

```
its('labels') { should include 'something-i-care-about' }
```

will throw a NoMethod error, looking for #include? on Nil::NilClass.

Consider setting it to be an empty array if there are no labels to parse.

Reply:
I just copied what was above to match style. Given the Boy Scout Rule you mentioned, I'll change both to be more Ruby like (e.g. `if j['names]`). I'll also set `labels` to an empty array if the field isn't present.